packages/widget/src/hooks/useRoutes.ts

```diff
@@ -93,7 +92,7 @@ export const useRoutes = ({ insurableRoute }: RoutesProps = {}) => {
     !Number.isNaN(slippage) &&
     hasAmount &&
     isToAddressSatisfied &&
-    isChainTypeSatisfied &&
+    // isChainTypeSatisfied &&
```

Comment:
I think this variable has been removed, maybe we can just delete this?

Reply:
Indeed 🙂